CHANGELOG.md

```diff
@@ -1,218 +1,219 @@
-* Fix: Schedules Direct was not returning the saved country in some cases.
-* Fix: Removed asterisks from password field when entering the password for Schedules Direct.
-* Fix: Fixed so that Ministry will not allow sleep while converting.
-* Fix: Allow mounting DVD iso images as non-root
-* Linux Placeshifter: Added AC3 support
-
-## Version 9.0.11 (2016-11-20)
-* Fix: Enable streams with valid PAT packets and invalid PMT packets to be able to be detected by the built in remuxer.
-* Fix: Linux tries a few more adapters when trying to get the primary server IP address.
-
-## Version 9.0.9 (2016-10-10)
-* Fix: GetSeriesID wasn't always returning a valid series ID
-* New: Added logic to Schedules Direct program categories to ensure Movie is the first category for programs that start with MV
-* Fix: Cleaned up the logic for determining when images from Schedules Direct should be in a Show or SeriesInfo object
-* Fix: Clarified in logging when we can't process anything currently because Schedules Direct is offline
-* Fix: Added random timeout when Schedules Direct token expires before getting a new token in case there are multiple SageTV servers using the same account
-
-## Version 9.0.8.429 (2016-09-27)
-* Fix: Fixed plugin bug that caused some upgraded plugins to be in a corrupted state
-* Fix: Fixed bug in the EPG license detection logic
-
-## Version 9.0.8 (2016-09-22)
-* New: Added Schedules Direct EPG support as a core BETA feature
-
-## Version 9.0.7 (2016-08-10)
-* New: Added SageTVPluginsDev.d directory support (See [SageTVPluginsDev README](SageTVPluginsDev.md))
-* New: Added direct JAR linking in SageTV Plugin Manifest (ie, no need to repackage library plugins as .zip files)
-
-
-#### Notes about incrementing versions for developers:
-
-* If you are the first to commit changes after a release, ensure that the following have been incremented beyond the last release:
-    * MICRO_VERSION in sage/Version.java
-* If you make any changes to stvs/SageTV7/SageTV7.xml, ensure that the following are updated in the STV:
-    * AddGlobalContext( "STVversionText", "August 12, 2017" )
-        * This should match the date of the commit.
-    * AddGlobalContext( "ThisSTVSetVersionNum", "2017081201" )
-        * This should match the date of the commit and if there was more than one commit the same day, the last two digits should be incremented.
-        * The format is YYYYMMDDVV.
-        * YYYY is the year.
-        * MM if the month number.
-        * DD is the day of the month.
-        * VV is the commit version for this date. This resets to 01 if the date changes.
-    * STVVersion [="9.1.7.0"]
-        * This should start with MAJOR_VERSION.MINOR_VERSION.MICRO_VERSION in sage/Version.java
-        * The last number should be incremented for each update of the STV for the MAJOR_VERSION.MINOR_VERSION.MICRO_VERSION SageTV release, starting with 0 for the first STV version of a new release.
+# Change Log

```

Comment:
There seems to be some kind of formatting change here (likely line endings) which marks this whole file has changed. Can you fix that please?

Reply:
Other than that this looks fine. Thanks!

Reply:
Egad!  Must be the line endings.  I have no idea why that happened; the same editor & settings was used to edit all 3 files in this commit.  Oh well, I've fixed it and then verified the change on my remote branch.  Please have a look and continue with the pull-request.   Thanks.